stage_descriptions/base-10-ny8.md

```diff
@@ -5,10 +5,11 @@ In this stage, you'll add support for the zero or one quantifier (`?`).
 The `?` quantifier makes the preceding character or character class optional, meaning it can appear either once or not at all.
 
 For example:
-- `dogs?` matches `"dog"` (zero `"s"`)
-- `dogs?` matches `"dogs"` (one `"s"`)
-- `dogs?` does not match `"dogss"` (two `"s"`)
-- `dogs?` does not match `"cat"` (doesn't match `"dog"` pattern)
+
+- `dogs?$` matches `"dog"` (zero `"s"`)
+- `dogs?$` matches `"dogs"` (one `"s"`)
+- `dogs?$` does not match `"dogss"` (two `"s"`)
+- `dogs?$` does not match `"cat"` (doesn't match `"dog"` pattern)
```

Comment:
I think adding `$` makes things a bit more confusing. Can we try to avoid the `$` usage?

Reply:
Yep, that works too!